client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -100,6 +102,7 @@ export interface PlanFeatures2023GridProps {
 	planTypeSelectorProps: PlanTypeSelectorProps;
 	// temporary: callback ref to scroll Odie AI Assistant into view once "Compare plans" button is clicked
 	observableForOdieRef: ( observableElement: Element | null ) => void;
+	selectedStorageOptions?: SelectedStorageOptionForPlans;
```

Comment:
`selectedStorageOptions` prop doesn't seem relevant here and should be removed.
The only relevant type for your case is `PlanFeatures2023GridType` below I think

(it's a little confusing, I know. parts that are being cleaned up in other work)

Reply:
Thanks for pointing this out! I removed the unnecessary type here https://github.com/Automattic/wp-calypso/pull/81340/commits/d5a8c5cf243569be2faf685300163ad81c12fd84